src/App.js

```diff
@@ -1,13 +1,16 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import NavBar from './components/NavBar';
 import Profile from './components/Profile';
+import Rockets from './components/Rockets';
+import './styles/rockets.css';
 
 function App() {
   return (
     <BrowserRouter>
       <NavBar />
       <Routes>
         <Route path="/profile" element={<Profile />} />
+        <Route path="/" element={<Rockets />} />
```

Comment:
[Optional] To depict the app's page hierarchy it is good to swap these two line.